vars/maven.txt

```diff
@@ -63,6 +70,13 @@ The <code>maven</code> variable can be used to read specific Maven information a
             <dd>the name of the JDK installation configured in Jenkins (optional)</dd>
         </dl>
     </dd>
```

Comment:
Documentation for `executeWithTool` is missing

Reply:
documentation added